dptb/data/dataset/_default_dataset.py

```
@@ -345,24 +413,16 @@ def __init__(
-                                get_eigenvalues,
-                                info=info)
-            else:
-                subdata = _TrajData.from_text_data(os.path.join(self.root, file), 
-                                get_Hamiltonian,
-                                get_overlap,
-                                get_DM,
-                                get_eigenvalues,
-                                info=info)
-            self.raw_data.append(subdata)
+            assert all(attr in info for attr in ["r_max", "pbc"])
+            pbc = info["pbc"] # not used?
```

Ruff (0.14.7)

Local variable `pbc` is assigned to but never used

Remove assignment to unused variable `pbc`

(F841)